.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hey, @IKraybill ! Nice to see the club is updating the website. Btw, is this `.idea` file a file from Visual Studio Code? If so, I suggest adding it to the `.gitignore` file.